pkg/cmd/step/create/step_create_task.go

```diff
@@ -1401,8 +1402,13 @@ func (o *StepCreateTaskOptions) runStepCommand(step *syntax.Step) error {
 
 	commandText := strings.Replace(step.GetFullCommand(), "\\$", "$", -1)
 
+	name := "/bin/sh"
+	if runtime.GOOS == "windows" {
+		name = "sh"
```

Comment:
I wonder whether it would be safe to always use "sh". This really should work on any OS. @abayer wdyt?

Reply:
Seems possible? I’m a little wary, but I may just be excessively conservative.

Reply:
Maybe we can use `cmd `.  WDYT?

Reply:
not sure about `cmd`. How would that work? I was just trying to about the special case for Windows. A simple 'sh' should work on macOS and Linux as well. But maybe @abayer is right and we should be a bit more conservative here.

Reply:
I don't have a way atm to verify that this actually works on Windows, so I guess we just see what happens.